wd_core.au3

```diff
@@ -1675,20 +1676,26 @@ EndFunc   ;==>__WD_CloseDriver
 ; #INTERNAL_USE_ONLY# ===========================================================================================================
 ; Name ..........: __WD_EscapeString
 ; Description ...: Escapes designated characters in string.
-; Syntax ........: __WD_EscapeString($sData)
-; Parameters ....: $sData - the string to be escaped
-; Return values..: Escaped string.
+; Syntax ........: __WD_EscapeString($sData[, $iOption = 0])
+; Parameters ....: $sData   - the string to be escaped
+;                  $iOption - [optional] Any combination of $JSON_* constants. Default is 0.
 ; Author ........: Danp2
 ; Modified ......:
-; Remarks .......:
+; Remarks .......: $JSON_MLREFORMAT will strip tabs and CR/LFs from a multiline string.
+;                  See $JSON_* constants in json.au3 for other $iOption possibilities.
 ; Related .......:
 ; Link ..........:
 ; Example .......: No
 ; ===============================================================================================================================
-Func __WD_EscapeString($sData)
-	Local $sRegEx = "([" & $_WD_ESCAPE_CHARS & "])"
-	Local $sEscaped = StringRegExpReplace($sData, $sRegEx, "\\$1")
-	Return SetError($_WD_ERROR_Success, 0, $sEscaped)
+Func __WD_EscapeString($sData, $iOption = 0)
+	If BitAND($iOption, $JSON_MLREFORMAT) Then
+		$sData = StringRegExpReplace($sData, '[\v\t]', '') ; Strip tabs and CR/LFs
```

Comment:
maybe here comment like this:
`; Strip tabs and CR/LFs - JavaScript linearization`

Reply:
> ; Remarks .......: $JSON_MLREFORMAT will strip tabs and CR/LFs from a multiline string.

I had considered adding the word Javascript here, but chose not to because there isn't a restriction to using this only with JS. Also, I don't believe that "linearization" is the correct word to describe this behavior. Please cite your sources if you believe otherwise.

Reply:
> I had considered adding the word Javascript here, but chose not to because there isn't a restriction to using this only with JS. 

ok

> Also, I don't believe that "linearization" is the correct word to describe this behavior. Please cite your sources if you believe otherwise.

none, this was mine poor attempt at using the English language

so lets stick with current comment